spec/migrations/20200912032913_add_ancestry_to_resource_pool_spec.rb

```diff
@@ -0,0 +1,181 @@
+require_migration
+
+describe AddAncestryToResourcePool do
+  let(:rel_stub) { migration_stub(:Relationship) }
+  let(:resource_pool_stub) { migration_stub :ResourcePool }
+  let(:resource_pool) { resource_pool_stub.create! }
```

Comment:
these are a bit much here.

Could you use specific name and put into the test blocks.
I really like the use of words like `parent_rel` and `root`.